DEVCONTAINER.md

```diff
@@ -0,0 +1,141 @@
+# Using Development Containers for development and testing
+
+You can set up a development environment for this project using [Development Containers](https://containers.dev/), aka devcontainers.
+
+There are different ways of working with devcontainers. Some are automated and integrated in IDE applications, while others are more manual and require CLI tools. This guide will use [Visual Studio Code](https://code.visualstudio.com) and assumes that it's installed already.
+
+Be aware that the containers will take significant space in your hard drive, in the region of 6GB.
+
+## Install Git
+
+You will need Git to clone (download) the code repository and push changes. How to install Git will depend on your operating system.
+
+### Ubuntu 24.04 LTS, Debian 13 (Trixie)
+
+```bash
+sudo apt-get update
+sudo apt-get install git-core
+```
+
+### Fedora
+
+```bash
+sudo dnf install git
+```
+
+### macOS
+
+For macOS, you will need [Xcode Command Line Tools](https://mac.install.guide/commandlinetools/); macOS 14 (Sonoma) or later; and some familiarity with Unix development via the Terminal.
+
+### Windows
+
+You can install Git using [WinGet](https://learn.microsoft.com/en-gb/windows/package-manager/winget/):
+
+```bash
+winget install --id Git.Git -e --source winget
+```
+
+Alternatively, see [other install options](https://git-scm.com/install/windows).
+
+## Install Docker
+
+Devcontainers run on top of Docker, which needs to be installed. Again, the methods will depend on your operating system.
```

Comment:
S: Change grammar
```suggestion
You will need to install Docker in order to run devcontainers. Again, the methods will depend on your operating system.
```

or
```suggestion
Installing Docker is required in order to run devcontainers. Again, the methods will depend on your operating system.
```

or alternatively
```suggestion
Devcontainers run on top of Docker, which needs to be installed on your operating system. Again, the methods will depend on your operating system.
```
(though this introduces duplicate text 'operating system'.)